src/snyk/snykCode/views/issueTreeProvider.ts

```diff
@@ -33,8 +33,18 @@ export class IssueTreeProvider extends ProductIssueTreeProvider<CodeIssueData> {
   getRunTestMessage = () => messages.runTest;
 
   // The title in the tree is taken from the title for vulnerabilities and from the message for quality rules
-  getIssueTitle = (issue: Issue<CodeIssueData>) =>
-    issue.additionalData.isSecurityType ? issue.title.split(':')[0] : issue.additionalData.message.split('.')[0];
+  getIssueTitle(issue: Issue<CodeIssueData>): string {
+    let issueTitle = issue.additionalData.isSecurityType
```

Comment:
Please fix the warnings from the linter :)

Reply:
Will do!